### PR TITLE
More markdown: event modal description

### LINKED ISSUE #1505: Markdown renderer needed for event modal
image

also in the admin interface -- renders the body nicely but not the title.

ping @margulies

### PR DESCRIPTION
**Pull request by yarikoptic**
This is how it would look like with this PR (edit: title is no longer rendered so must not be markdown)

image

in comparison to 

image

Closes #1505

### CODE REVIEW

File: src/components/molecules/InformationCard/InformationCard.tsx [hunk 1]
```diff
@@ -15,7 +17,9 @@ const InformationCard: React.FC<InformationCardProps> = ({
   className,
 }) => (
   <div className={classNames("information-card-container", className)}>
-    <h4 className="title">{title}</h4>
+    <h4 className="title">
```
Comment (0xdevalias) [inline src/components/molecules/InformationCard/InformationCard.tsx:20]: Pretty sure rendering in a title tag like this will cause issues similar to the p in p issue

File: src/components/molecules/ScheduleEvent/ScheduleEvent.tsx [hunk 2]
```diff
@@ -81,7 +82,9 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
     <>
       <div className={containerClasses} onClick={showEventModal}>
         <div className="ScheduleEvent__info">
-          <div className="ScheduleEvent__title">{event.name}</div>
+          <div className="ScheduleEvent__title">
```
Comment (0xdevalias) [inline src/components/molecules/ScheduleEvent/ScheduleEvent.tsx:85]: Why do we want to render markdown in titles?

File: src/components/organisms/EventModal/EventModal.scss [hunk 3]
```diff
@@ -24,13 +24,6 @@ $EventModal--padding-top: 40px;
     opacity: 0.8;
   }
 
-  &__description {
```
Comment (0xdevalias) [inline src/components/organisms/EventModal/EventModal.scss:27]: Why was this removed?

File: src/components/organisms/EventModal/EventModal.tsx [hunk 4]
```diff
@@ -68,15 +70,17 @@ export const EventModal: React.FC<EventModalProps> = ({
...
+          <RenderMarkdown text={event.name} />
+        </h4>
         <span className="EventModal__subtitle">
           by {event.host} in{" "}
           <button className="button--a" onClick={goToEventLocation}>
             {eventLocationToDisplay}
           </button>
         </span>
 
-        <p className="EventModal__description">{event.description}</p>
```
Comment (0xdevalias) [inline src/components/organisms/EventModal/EventModal.tsx:79]: I assume this was done to avoid the ‘p in p’ error, but I don’t think the solution here is what we want to be doing. Without looking too deeply, I would assume that changing the p to a dig and nesting RenderMarkdown within that would have been the needed solution.

File: src/components/organisms/EventModal/EventModal.tsx [hunk 5]
```diff
@@ -68,15 +70,17 @@ export const EventModal: React.FC<EventModalProps> = ({
   return (
     <Modal show={show} onHide={onHide} className="EventModal">
       <div className="EventModal__content">
-        <h4 className="EventModal__title">{event.name}</h4>
+        <h4 className="EventModal__title">
```
Comment (0xdevalias) [inline src/components/organisms/EventModal/EventModal.tsx:73]: As above, why do we want to render markdown in titles?

**Review by 0xdevalias — COMMENTED** [review]
High level quick pass of comments/questions (from my phone)

File: src/components/molecules/InformationCard/InformationCard.tsx (hunk 1 above)
Comment (yarikoptic) [reply src/components/molecules/InformationCard/InformationCard.tsx]: didn't see the problem, but indeed decided to not bother about titles (left sample event though with markdown for testing).  Removed them all.  Now this PR has only markdown rendering for event description

File: src/components/organisms/EventModal/EventModal.scss (hunk 3 above)
Comment (yarikoptic) [reply src/components/organisms/EventModal/EventModal.scss]: because was not used in that version. now brought back (although I do not see any effect of it when used as `div`)

File: src/components/organisms/EventModal/EventModal.tsx (hunk 5 above)
Comment (yarikoptic) [reply src/components/organisms/EventModal/EventModal.tsx]: no more

**Review by mike-lvov — APPROVED** [review]
LGTM
